Makefile

```diff
@@ -29,13 +31,43 @@ INTERNAL_TMP_DIR=/tmp/devworkspace-controller
 BUMPED_KUBECONFIG=$(INTERNAL_TMP_DIR)/kubeconfig
 RELATED_IMAGES_FILE=$(INTERNAL_TMP_DIR)/environment
 
+ifeq ($(shell kubectl api-resources --api-group='route.openshift.io' | grep -o routes),routes)
+PLATFORM := openshift
+else
+PLATFORM := kubernetes
+endif
+
 # minikube handling
-ifeq ($(shell $(TOOL) config current-context),minikube)
-	ROUTING_SUFFIX := $(shell minikube ip).nip.io
-	TOOL := kubectl
+ifeq ($(shell kubectl config current-context),minikube)
+export ROUTING_SUFFIX := $(shell minikube ip).nip.io
+endif
+
+
+# Bootstrapped by Operator-SDK v1.1.0
+# Current Operator version
+VERSION ?= 0.0.1
+# Default bundle image tag
+BUNDLE_IMG ?= controller-bundle:$(VERSION)
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+CRD_OPTIONS ?= "crd:trivialVersions=true"
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
 endif
 
-all: help
+all: manager
```

Comment:
What do you think if we leave help as default rule?

Reply:
:+1: This has already tripped me up me a few times during development :)